playbooks/ci-full/templates/heat_stack.yml

```diff
@@ -0,0 +1,230 @@
+heat_template_version: 2013-05-23
```

Comment:
This version is from the template in sitecontroller code. And that is a valid version number mentioned in heat HOT specification: http://docs.openstack.org/developer/heat/template_guide/hot_spec.html#id1.
I think that version can fulfill our requirement, if we encounter an issue with the version in the future, we can consider change to a later one then. It works well for now.
fyr, HOT versions have these options: 2013-05-23, 2014-10-16, 2015-04-30, etc